src/app/shared/routing/group-route.ts

```diff
@@ -15,22 +17,49 @@ export interface GroupRouteError {
   id?: string;
 }
 
-export function groupRoute(id: GroupId, path: string[]): GroupRoute {
-  return { contentType: 'group', id: id, path };
+
+export type GroupLike =
+  | { id: Group['id'], type?: string }
```

Comment:
Do you need | before first type?

Reply:
No, this notation is mainly for further iterations and/or readability so you can compare the union-ed types in a glimpse
But actually the type probably won't change

Should I still change this type and make it one-line ?